server/bin/pbench-backup-tarballs.py

```diff
@@ -491,7 +491,7 @@ def main():
         return 2
 
     try:
-        config = PbenchConfig(cfg_name)
+        config = PbenchConfig(cfg_name, "pbench-backup-level")
```

Comment:
Do not modify this. Instead modify line 499ff below (similarly for the other scripts, pbench-verify-backupt-tarballs.py and pbench-index.py).

After you get the logger, query the config object for the logging level for this app. If you find one, 
then set the logger's logging level to what you found - otherwise leave it alone:

      logger = get_pbench_logger(_NAME_, config)
      try:
           logging_level = config.get(....)
           logger.setLevel(logging_level)
       except NoSectionError, NoOptionError:
            pass

Reply:
ok